typescript/src/schema/api.ts

```diff
@@ -32,19 +32,28 @@ export class API {
   hostName?: string;
   port?: string;
   mainServiceName?: string;
+  //This field is for users passing roper module name for system test.
```

Comment:
`proper` (also, please add a space after `//`)

Reply:
thank you!

Reply:
still unresolved :)

Reply:
still unresolved :) Still no space, and `roper`

Reply:
sorry, thought it was another file!